src/Qdrant.Client/QdrantClient.cs

```diff
@@ -377,15 +397,17 @@ public class QdrantClient : IDisposable
 		QuantizationConfig? quantizationConfig = null,
 		string? initFromCollection = null,
 		TimeSpan? timeout = null,
+		ShardingMethod? shardingMethod = null,
```

Comment:
Consider keeping `timeout` and `cancellationToken` as the last two parameters

Reply:
fixed